src/main/java/de/tum/in/www1/artemis/service/ResultService.java

```diff
@@ -427,4 +427,24 @@ public void notifyUserAboutNewResult(Result result, Long participationId) {
     private void notifyNewResult(Result result, Long participationId) {
         websocketMessagingService.sendMessage("/topic/participation/" + participationId + "/newResults", result);
     }
+
+    /**
+     * Create a new example result for the provided submission ID.
+     *
+     * @param submissionId The ID of the example submission for which a result should get created
```

Comment:
The comment is a bit misleading. Is it the id of the submission or the id of the example submission?
I guess it is the id of a submission that belongs to an example submission right?

I guess we should also emphasize this in the comment at the top, that the submission must be connected to an example submission, otherwise this call throws an illegal argument exception. Otherwise another developer might be tempted to use this method.

I even think you should rename the method to `createNewExampleResultForSubmissionWithExampleSubmission` or something similar